static/translations/messages_en.json

```diff
@@ -17,6 +17,7 @@
   "Pick account": "Pick account",
   "Welcome": "Welcome",
   "Enter your Zulip server URL:": "Enter your Zulip server URL:",
+  "e.g. zulip.example.com": "e.g. zulip.example.com",
```

Comment:
This helper text isn't actually helpful – it's redundant with the existing placeholder-slash-autocomplete text.

Reply:
It's a different example, though; my thought is to make this bit of UI less zulipchat.com-centric.

Reply:
From a bit of in-person discussion: AIUI you think this example will actually be confusing, but I'm not persuaded.

Given that we want to get this set of changes out quickly, I'm going to go ahead and merge, and then see about a release. This is certainly a screen where it makes sense to iterate further on the UI, though; we can always change it again next.